packages/test/test-end-to-end-tests/src/test/blobsisAttached.spec.ts

```diff
@@ -14,10 +14,14 @@ import {
 	ITestObjectProvider,
 	DataObjectFactoryType,
 	createAndAttachContainer,
+	timeoutPromise,
 } from "@fluidframework/test-utils";
 import { describeNoCompat } from "@fluid-internal/test-version-utils";
 import { stringToBuffer } from "@fluidframework/common-utils";
 import { IFluidHandle } from "@fluidframework/core-interfaces";
+import { ContainerRuntime } from "@fluidframework/container-runtime";
+// eslint-disable-next-line import/no-internal-modules
+import { PendingLocalState } from "@fluidframework/container-runtime/dist/test";
```

Comment:
The pull request has been merged now, but I have a slight concern regarding directly importing something from the compiled output located in the `/dist`. One possible approach could be to create an `index.ts` file under `container-runtime/test` and include `export { PendingLocalState }` inside it. Then, you can update the `index.ts` file under `container-runtime` to include `export { PendingLocalState } from './test'`. Just a personal suggestion.